views/article.html.twig

```diff
@@ -53,10 +53,31 @@
 					  {% endgimme %}
 					{% endif %}
 
-				<figure class="article__image">
-        	<img src="{{ rend1250 }}" alt="{{ imgCaption }}"  />
-        	<figcaption>{{ imgCaption }}</figcaption>
-				</figure>
+{% if gimme.article.featureMedia %}
+
+{% gimme rendition with { 'media': gimme.article.featureMedia, 'name': '1250x600', 'fallback': 'original' } %}
+	{% set rend1250 = path(gimme.rendition) %}
+{% endgimme %}
+
+{% else %}
+
+{% gimmelist media from articleMedia|limit(1) with {'article': article} %}
+{% set imgCaption = media.body %}
+  {% if media.renditions is iterable %}   
```

Comment:
replace it with:
```
{% gimme rendition with { 'name': '1250x600', 'fallback': 'original' } %}
        {% set rend1250 = path(rendition) %}
{% endgimme %}```

more here: http://superdesk-web-publisher.readthedocs.io/en/latest/templates_system/article_media.html#image-renditions